pkg/nodeps/utils.go

```diff
@@ -135,3 +136,19 @@ func GrepStringInBuffer(buffer string, needle string) []string {
 	matches := re.FindStringSubmatch(buffer)
 	return matches
 }
+
+// PathExplode returns an array of all possible paths separated by slashes out
+// of a single one.
+// i.e. path/to/file will return {"path", "path/to", "path/to/file"}
+func PathExplode(path string) []string {
+	var paths []string
+	var partial string
+	for _, p := range strings.Split(path, "/") {
```

Comment:
You can't be using strings.Split() on filepaths. Instead use the go `filepath` package to do what you need. Remember... Windows. And weird embedded thigns of other types.

And shouldn't this be in the fileutil package, not utils?

Reply:
The package I haven't thought about, if you think it has to go there, I'll move it. About slashes, you seem to use slashes on other places and I added the documentation that this is expecting slashes. It's more a string function that a path function. I would like to understand this better before spending more time unless it's really required.